src/Provider/PointCloudProvider.js

```diff
@@ -141,8 +143,24 @@ function addPickingAttribute(points) {
     return points;
 }
 
+function bboxToExtent(view, bbox) {
```

Comment:
this only depends on the crs -> `bboxToExtent(crs, bbox)`. I would prefer something like `extent.fromBox3(box3, crs)`.
Should not it share the same implementation as in the bbox case for 3dtiles (except with no transform) ?

Reply:
I'm reluctant to abstract / share code too early. So I prefer to keep the 2 separate implementations.

Maybe in some time we may want to merge them (or move them up in Extent) - or maybe not for reasons that we're not aware of now.

Reply:
I see your point, but the downside of your approach is that you end up coding twice the box3 to extent conversion with 2 different implementations and no tests...

Why is the hard-coded crs test required in one case and not the other?

Reply:
Done - and removed the 4978 -> 4326 conversion, since it loses one information (z)